static/docs/commands-reference/cache.md

```diff
@@ -0,0 +1,65 @@
+# cache
```

Comment:
The general structure we should follow for a command reference is the one in `run.md`. We should though come up with some good readable structure for a command with subcommands - I have no idea so far, probably it makes more sense to do a separate file for `cache dir`, etc - it will be cleaner.